### DISCUSSION TITLE
在mac的浏览器获取el-table的滚动条高度永远都是table的高度

### DISCUSSION THREAD

**Discussion by yaolunmao** [element-plus/element-plus#6422] [Q&A]
在windows就是实际内容的高度，而且在mac里面滚动之后获取scrollTop一直是0，windows就正常，有人遇到过吗

**Comment by msidolphin** [comment]
你获取滚动条高度目前是什么？你需要监听.el-scrollbar__wrap的滚动事件

**Reply by yaolunmao** [reply]
使用.addEventListener('scroll')确实可以 但是我的需求是点击一个坐标按钮 让表格滚到相对应的位置 我现在不管怎么滚动表格里面的数据拿到的scrollTop都一直是0

**Reply by msidolphin** [reply]
明白，我们可以提供滚动接口，麻烦提一个feature request

**Reply by yaolunmao** [reply]
提好了 谢谢您
